src/Signer/Blake2b.php

```diff
@@ -26,7 +27,10 @@ public function sign(string $payload, Key $key): string
             throw InvalidKeyProvided::tooShort(self::MINIMUM_KEY_LENGTH_IN_BITS, $actualKeyLength);
         }
 
-        return sodium_crypto_generichash($payload, $key->contents());
+        $hash = sodium_crypto_generichash($payload, $key->contents());
```

Comment:
This too... it seems like something we could improve on PHPStan stubs (AFAIK we'd always have a string with size of `$length` or an exception).

Reply:
Yeh, but IMO out of scope for this PR, highlighted by future lockfile changes

Reply:
What I meant is `@phpstan-ignore-line` would be simpler than adding the assertions...

Reply:
Eh, iz fine with the assertion: we keep the original understanding/intent, and get better error messages later on

Reply:
FWIW, meanwhile: https://github.com/vimeo/psalm/pull/8681

Reply:
https://github.com/phpstan/phpstan-src/pull/1980

Reply:
Also https://github.com/phpstan/phpstan-src/pull/1981

Reply:
Cool, let's clean things up later on then 👍